hack/generator/pkg/astmodel/identifier_factory.go

```diff
@@ -88,6 +96,14 @@ func createRenames() map[string]string {
 	}
 }
 
+// These are words reserved by go
+func createReservedWords() map[string]struct{} {
+	return map[string]struct{}{
+		"package": {},
```

Comment:
Should expand this list probably

Reply:
we might as well have this be a shared map as we will never mutate it

Reply:
Identifier Factory is a singleton, so I think it's a moot point.

Reply:
is it?

Reply:
If by singleton you mean: `IdentifierFactory` is only constructed once, that is true.

I've left this as is as it matches the renames collection.